sdk/servicebus/service-bus/test/utils/envVarUtils.ts

```diff
@@ -108,6 +118,36 @@ function getEnvVarValue(name: string, forBrowser?: boolean): string | undefined
     case EnvVarKeys.TOPIC_FILTER_DEFAULT_SUBSCRIPTION_NAME:
       defaultValue = "topic-filter-default-subscription";
       break;
+    case EnvVarKeys.MANAGEMENT_QUEUE_1:
```

Comment:
I feel like this entire switch statement should be replaced with an object mapping names to default values. Then the function can just return `defaultValue[name]`

Reply:
Agree, simplified!